extension/scripts/background.js

```diff
@@ -12,6 +12,45 @@ if (localStorage.getItem('enabled') === null) {
 	enabled = localStorage.getItem('enabled') === 'true';
 }
 
+let blackListPromise;
+(function updateBlacklist () {
+	blackListPromise = fetch(
+		'http://bertha.ig.ft.com/view/publish/gss/1_f-cR8VbcujCsjxTLyKVRcG9yp_qUbT0-FrdmwBp0OM/blacklist'
+	).then(response => response.json());
+
+	// Once an hour check for new black listed sites.
+	setTimeout(updateBlacklist , 3600 * 1000);
+}());
+
+function checkIsNotBlackListed (url) {
+
+	function escapeRegExp (str) {
```

Comment:
This function is recreated every time `checkIsNotBlacklisted` is called. Could we pull it out of this function definition?